### ISSUE TITLE
Hiding the `.statusbar-overlay` on landscape orientation

### ISSUE THREAD

**Issue by iBinJubair** [nolimits4web/Framework7#1365]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: 1.5.3
* Platform and Target: iOS

### What you did
I added this code to css file.
`@media screen and (orientation:landscape) {
	html.with-statusbar-overlay .framework7-root { padding-top: 0; }
	html.with-statusbar-overlay .statusbar-overlay { height: 0 ; }
	html.with-statusbar-overlay .framework7-root>.photo-browser, html.with-statusbar-overlay body>.photo-browser { top: 0; height:100%; }
}`

### Expected Behavior / Actual Behavior
If you add the website to home screen and open the it as a web app,
iOS(iPhone) will hide status bar on landscape orientation. The `.statusbar-overlay` div will still be shown! The above code should hide it.

**Comment by valnub** [comment]
So, you already solved your problem?

**Comment by iBinJubair** [comment]
@valnub Yes, I just share what I found. So, someone can add it to master.

**Comment by valnub** [comment]
Can you please post a pull-request instead? https://help.github.com/articles/about-pull-requests/

thanks
